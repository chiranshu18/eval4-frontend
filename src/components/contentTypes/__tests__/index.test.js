import ContentTypes from '../index';
import { render , screen} from '@testing-library/react';
import React from 'react';

describe('Content Types', () => {
  it('Should render correctly', () => {
    const {asFragment} = render(<ContentTypes />);
    expect(asFragment()).toMatchSnapshot();
  });
});