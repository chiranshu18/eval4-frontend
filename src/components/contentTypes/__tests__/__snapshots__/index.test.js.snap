// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content Types Should render correctly 1`] = `
<DocumentFragment>
  <div
    class="content-types"
  >
    <div
      class="add-new-content-type-card"
    >
      <p
        class="add-new-content-type-card-text"
      >
        New Types+
      </p>
    </div>
  </div>
</DocumentFragment>
`;
